AppCenterData/AppCenterData/MSReadOptions.m

```diff
@@ -1,8 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#import "MSData.h"
```

Comment:
Please run formatter again.

Reply:
formatter doesn't sort it correctly. i re-ran and let it to  decide ;)